src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,40 @@
+import React = require('react');
+
+export interface CheckBoxProps {
+    text?: string;
+    value?: boolean;
+    boxIcon?: string;
+    tickIcon?: string;
+    onChange?: (value: boolean) => any;
+}
+
+export const CheckBox: React.SFC<CheckBoxProps> = (props) => (
+    <div data-automation-id="CHECKBOX_ROOT"
+         onClick={(event) => executeClickHandler(props.onChange!, !props.value)}>
+        <img data-automation-id="CHECKBOX_BOX"
+             src={props.boxIcon}
+             style={{width: '1em', height: '1em'}}/>
+        <img data-automation-id="CHECKBOX_TICKMARK"
+             src={props.tickIcon}
+             style={{
+                 width: '1em',
+                 height: '1em',
+                 display: props.value ? '' : 'none',
```

Comment:
The motivation for using `display: none` instead of conditional rendering was to preload the image and have it in the cache for when the checkbox is toggled on.

On the other hand, the images are probably going to be combined into a sprite anyway, so I don't know. What's our preferred approach?